app/routes/bdb/components/BdbDetail.js

```diff
@@ -53,6 +53,10 @@ export default class BdbDetail extends Component<Props, State> {
     eventsToDisplay: 3
   };
 
+  componentDidMount() {
+    window.scrollTo(0, 0);
+  }
```

Comment:
Is doing this fine? While searching for a solution it seemed like a lot of projects added this functionality to Router so that it happened for all navigations, but I don't know if we want that.

Reply:
Hmm. Looks like it works well here, but imo it should not be used everywhere.. It is nice to click back, and retain your scrolling position.